micrometer-core/src/main/java/io/micrometer/core/instrument/binder/system/DiskSpaceMetrics.java

```diff
@@ -33,28 +35,40 @@
  *
  * @author jmcshane
  * @author Johnny Lim
+ * @author Chris Bono
  * @since 1.8.0
  */
 @NonNullApi
 @NonNullFields
 public class DiskSpaceMetrics implements MeterBinder {
+
     private final Iterable<Tag> tags;
-    private final File path;
-    private final String absolutePath;
+    private final List<File> paths;
```

Comment:
We could be less specific than `List` since we don't care about order. How about `Collection`?